rotkehlchen/chain/evm/decoding/decoder.py

```diff
@@ -793,6 +794,29 @@ def _decode_transaction(
         if process_swaps:
             events = self._process_swaps(transaction=transaction, decoded_events=events)
 
+        if (
+                tx_receipt.tx_type == 4 and
+                transaction.from_address == transaction.to_address and
+                transaction.authorization_list is not None
+        ):  # if the event is an eip-7702 transaction, we need to add an informational event.
+            if (delegated_address := transaction.authorization_list[-1].delegated_address) == ZERO_ADDRESS:  # noqa: E501
+                notes = 'Successfully revoked eip-7702 transaction authorization'
```

Comment:
my problem with having eip-7702 is that people forget what those are and also most of the users don't know what it is so I find it confusing to have in the notes. I belive they key information here is that you either revoked or delegatd